ceph-releases/luminous/ubuntu/16.04/daemon/osd_scenarios/osd_disk_activate.sh

```diff
@@ -30,7 +30,7 @@ function osd_activate {
   if [[ ${OSD_DMCRYPT} -eq 1 ]] && [[ ${OSD_FILESTORE} -eq 1 ]] && [[ ${OSD_BLUESTORE} -eq 0 ]]; then
     DATA_UUID=$(get_part_uuid "$(dev_part "${OSD_DEVICE}" 1)")
     LOCKBOX_UUID=$(get_part_uuid "$(dev_part "${OSD_DEVICE}" 5)")
-    JOURNAL_PART=$(ceph-disk list "${OSD_DEVICE}" | awk '/ceph journal/ {print $1}') # This is a privileged container so 'ceph-disk list' works
+    JOURNAL_PART=$(ceph-disk list "${OSD_DEVICE}" | grep journal | sed -r 's/^.*\s([^ ]+)$/\1/') # This is a privileged container so 'ceph-disk list' works.
```

Comment:
👎  when using dmcrypt the output is different... so this command is correct;

Reply:
let's use json output then

Reply:
or `awk`? :p